pkg/sysregistriesv2/testdata/pull-sources-from-reference.conf

```diff
@@ -12,10 +12,33 @@ insecure = true
 [[registry]]
 prefix = "registry-b.com/foo"
 location = "registry-b.com/bar"
-mirror-by-digest-only = true
```

Comment:
I am a afraid of a potential (future) regression if alter this existing test.  We should leave this as is but probably add a new one to test for `mirror-by-digest-only = {true,false}` and mirrors set `digest-only={true,false}` (mixed)